client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java

```diff
@@ -67,10 +67,11 @@ public interface CommonItemConfig {
     Optional<String> additionalRequestArgs();
 
     /**
-     * Defines if the methods should return {@link jakarta.ws.rs.core.Response} or a model. Default is {@code false}.
+     * Defines if the methods should return {@link jakarta.ws.rs.core.Response},
+     * {@link org.jboss.resteasy.reactive.RestResponse} or a model. Default is {@code false}.
```

Comment:
What's the default now? `false`? What does this mean? We should clarify this a little bit better. 

> By default, it returns the model in the specification

Or something like that.